src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -58,12 +59,17 @@ export function useTextureData(): TextureDataState {
       return;
     }
 
+    // Keep existing texture data, if any
+    mergeState({ loading: true });
+
+    // Prepare transferable buffer to avoid cloning values array
+    const typedValues = Float64Array.from(values);
+
     (async () => {
-      mergeState({ loading: true }); // keep existing texture data, if any
       mergeState({
         loading: false,
         textureData: await proxy.computeTextureData(
-          values,
+          transfer(typedValues, [typedValues.buffer]),
```

Comment:
That's the `postMessage` syntax to transfer a buffer https://devdocs.io/dom/worker/postmessage